### PR TITLE
Update logi-options-plus from latest to 1.30.0.7349

### PR DESCRIPTION
**Pull request by Moulick**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by Moulick** [comment]
I cannot find any way to implement a live check from the logi options plus. Should we still put a version? We can still bump the versions manually whenever we find a new version

File: Casks/logi-options-plus.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask "logi-options-plus" do
-  version "1.22.5550"
+  version "1.22.0.5550"
```
Comment (Moulick) [inline Casks/logi-options-plus.rb:2]: umm, why added a `0`? 
image

File: Casks/logi-options-plus.rb [hunk 2]
```diff
@@ -8,35 +8,40 @@
   desc "Software for Logitech devices"
   homepage "https://www.logitech.com/en-us/software/logi-options-plus.html"
 
+  livecheck do
+    url :url
+    strategy :extract_plist
```
[the remark this answers is not in the document]
Comment (miccal) [reply Casks/logi-options-plus.rb:13]: I think that is what already happens, but @samford may know better.

File: Casks/logi-options-plus.rb (hunk 1 above)
Comment (miccal) [reply Casks/logi-options-plus.rb:2]: Because that is the `version` shown in the `plist`:

Screen Shot 2022-09-20 at 16 33 29

Comment (Moulick) [reply Casks/logi-options-plus.rb:2]: ok got it, thanks for fixing 😄

File: Casks/logi-options-plus.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (Moulick) [reply Casks/logi-options-plus.rb:13]: yeah, I checked locally, it does extract the zip. It's still not able to fetch the version from the plist while the same works for `fnatic-op`. The one thing I see missing from `Info.plist` of `logi-options-plus` compared to `fnatic-op` is `<key>CFBundleShortVersionString</key>` while both do contain `<key>CFBundleVersion</key>`

**Comment by bevanjkay** [comment]
At the moment `:extract_plist` won't check installer artifacts. I started a PR to add this functionality https://github.com/Homebrew/brew/pull/13658 - I just haven't been able to get around to finishing it off just yet.

**Comment by Moulick** [comment]
> At the moment `:extract_plist` won't check installer artifacts. I started a PR to add this functionality [Homebrew/brew#13658](https://github.com/Homebrew/brew/pull/13658) - I just haven't been able to get around to finishing it off just yet.

What I don't get is even without your improvement, why does it `:extract_plist` work on `fnatic-op` but not on `logi-options-plus`? Both have an URL that downloads a zip and both have an Info.plist in the same location 🤔

**Comment by bevanjkay** [comment]
@Moulick It's working on `fnatic-op` because it uses an `app` stanza which is currently supported by the `:extract_plist` livecheck strategy.

**Comment by Moulick** [comment]
Can we add an `in progress` label to prevent closing?

**Comment by MTCoster** [comment]
Is it worth marking this as `auto_updates` here or should I open a separate PR for that?

**Comment by Moulick** [comment]
As this just downloads the installer and the installer in turn downloads the actual app, should this have `auto_updates`?

**Comment by MTCoster** [comment]
If it's not marked `auto_updates`, won't `brew upgrade` reinstall it unnecessarily?

**Comment by Moulick** [comment]
yeah, I just read https://docs.brew.sh/Cask-Cookbook#:~:text=parse.rb)-,auto_updates,-no , I'll add the auto_updates

**Comment by Moulick** [comment]
Should probably remove the stable label again?

**Comment by Moulick** [comment]
Whoo!! After merging https://github.com/Homebrew/brew/pull/13658 by @bevanjkay, this finally works. Thanks a lot @bevanjkay, much appreciated 😃

**Review by p-linnane — APPROVED** [review]

**Pull request closed without merging**